R/nicenumericcuts.R

```diff
@@ -111,28 +111,24 @@ NiceNumericCuts <- function(input.data,
 
     # Protect against empty strings supplied
     # by Displayr UI
-    if (!is.null(equal.intervals.start)) {
-        if(nzchar(equal.intervals.start)) {
-            equal.intervals.start <- as.numeric(equal.intervals.start)
-        } else {
-            equal.intervals.start <- NULL
-        }    
-    }
+    if(nzchar(equal.intervals.start)) {
+        equal.intervals.start <- as.numeric(equal.intervals.start)
+    } else {
+        equal.intervals.start <- NA
+    }    
```

Comment:
Purely stylistic choice but could change this to be defined in a single line
```suggestion
    equal.intervals.start <- if (nzchar(equal.intervals.start)) as.numeric(equal.intervals.start) else NA
```
(and similarly for the the `.end` variant)
EDIT: Looks like github wants to do a soft line wrap when viewing browser but I'm comfortable with a longer line.